frontend/src/components/common/button/button.tsx

```diff
@@ -10,10 +10,11 @@ import { getValidClasses } from 'helpers/helpers';
 import styles from './styles.module.scss';
 import { Link } from '../link/link';
 import deleteIcon from 'assets/img/delete-icon.svg';
+import editIcon from 'assets/img/edit-icon.svg';
 
 type Props = {
   label: string;
-  to?: AppRoute;
+  to?: AppRoute | string;
```

Comment:
```suggestion
  to?: AppRoute;
```
cast a type where you need it (`anyString as AppRoute`)